src/components/app/MenuButtons.js

```diff
@@ -578,6 +578,21 @@ type MenuButtonsProps = ConnectedProps<
   MenuButtonsDispatchProps
 >;
 
+function _mapMetaInfoExtensionNames(data: any) {
+  const extensionData = data;
+  const extensionList = extensionData.map(d => (
+    <li className="metainfoListItem" key={d}>
```

Comment:
Small style nit: This and all the class names should be camel cased as `metaInfo...` rather than `metainfo...`